java/arcs/sdk/android/storage/ServiceStore.kt

```diff
@@ -142,14 +150,19 @@ class ServiceStore<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
     override fun off(callbackToken: Int) {
         val service = checkNotNull(storageService)
         runBlocking {
-            channel.send { service.unregisterCallback(callbackToken) }
+            send { service.unregisterCallback(callbackToken) }
         }
+        initChannel()
```

Comment:
Shouldn't we wait until we've `off`'d all the callback tokens before we re-init the channel?